examples_tests/06.MeshLoaders/main.cpp

```diff
@@ -104,10 +104,64 @@ int main()
 
     smgr->setActiveCamera(camera);
 
+    auto createDefaultFBOForScreenshoting = [&]()
```

Comment:
useful as a function/method of the extension

Reply:
I will provide it, but first I need to figure out something I asked you about